smarty/data/odds_files/substitutions.smarts

```diff
@@ -0,0 +1,9 @@
+% Substitution definitions
+% Format:
+% <SMARTS> <replacement-string>
+% halogens
+[#7!-1,#8,#16]              ewg2
+[#7!-1,#8!-1,#16!-1,$halogen]     ewg1
```

Comment:
These are copied from @cbayly13 
Traditionally: ewg1 are single bonded and ewg2 are double bonded. So oxygen, nitrogen, and sulfur are in both, but the halogens can only be single bonded.